### ISSUE TITLE
Traffic not routed with docker 1.7

### ISSUE THREAD

**Issue by hadim** [jwilder/nginx-proxy#189]
Did someone make it work with docker 1.7 ?

I followed the instruction but my traffic get stuck inside the proxy and display the nginx welcome page... while I correctly setup my container with `-e VIRTUAL_HOST=mydomain.fr`.

```
$ d version
Client version: 1.7.0
Client API version: 1.19
Go version (client): go1.4.2
Git commit (client): 0baf609
OS/Arch (client): linux/amd64
Server version: 1.7.0
Server API version: 1.19
Go version (server): go1.4.2
Git commit (server): 0baf609
OS/Arch (server): linux/amd64
```

**Comment by hadim** [comment]
The nginx docker returns this : 

```
[localhost] local: docker run -ti --name nginx-proxy -p 80:80 jwilder/nginx-proxy
forego     | starting nginx.1 on port 5000
forego     | starting dockergen.1 on port 5100
dockergen.1 | 2015/06/26 18:26:14 error listing containers: dial unix /tmp/docker.sock: no such file or directory
dockergen.1 | 2015/06/26 18:26:14 Unable to ping docker daemon: dial unix /tmp/docker.sock: no such file or directory
dockergen.1 | 2015/06/26 18:26:24 Unable to ping docker daemon: dial unix /tmp/docker.sock: no such file or directory
dockergen.1 | 2015/06/26 18:26:34 Unable to ping docker daemon: dial unix /tmp/docker.sock: no such file or directory
dockergen.1 | 2015/06/26 18:26:44 Unable to ping docker daemon: dial unix /tmp/docker.sock: no such file or directory
```

**Comment by hadim** [comment]
I have the feeling it's related to #114. Can someone confirm ?

Any idea to fix it ? I am using lxc-docker-1.7 package on Linux Mint.

**Comment by hadim** [comment]
Aaaaahhhh I am sorry for the noise. I forgot to add `-v /var/run/docker.sock:/tmp/docker.sock:ro` to the nginx proxy launch command...

It works like a charm. 

Thank you for this really awesome container !
